sunpy/util/types.py

```diff
@@ -0,0 +1,10 @@
+"""
+A collection of ABCs to use to shortcut having to import things.
```

Comment:
I think this docstring should be a little more descriptive, including elaborating on "things".